optimum/bettertransformer/models/encoder_models.py

```diff
@@ -17,7 +17,6 @@
 from .base import BetterTransformerBaseLayer
 
 
-class TapasLayerBetterTransformer(BetterTransformerBaseLayer):
     def __init__(self, tapas_layer, config):
```

Comment:
I meant to remove the class `TapasLayerBetterTransformer` in full 😅 It is not needed in the end since `BertLayerBetterTransformer` can handle Tapas.

Reply:
I think I removed `class TapasLayerBetterTransformer`. Is there anything to be adjusted in this file?

Reply:
On your branch, with optimum dev installed (`pip install -e .` in the optimum top folder), can you try to run the script @younesbelkada provided you and see what happens?

```python
from transformers import AutoModel
from optimum.bettertransformer import BetterTransformer

model_id = "hf-internal-testing/tiny-random-TapasModel"
model = AutoModel.from_pretrained(model_id)
bt_model = BetterTransformer.transform(model)
```